tests/output.rs

```diff
@@ -1,228 +1,118 @@
 use assert_cmd::prelude::*;
-use once_cell::sync::Lazy;
-use predicates::{prelude::*, BoxPredicate};
-use std::{
-    env,
-    ffi::OsStr,
-    path::{Path, PathBuf},
-    process::Command,
-};
+use predicates::prelude::*;
+use std::{env, path::PathBuf, process::Command};
 
-struct Location {
-    line: usize,
-    #[allow(dead_code)]
-    column: usize,
+#[macro_export]
+macro_rules! regex {
+    ($re:literal $(,)?) => {{
+        static RE: once_cell::sync::OnceCell<regex::Regex> = once_cell::sync::OnceCell::new();
```

Comment:
Unrelated, but I think that part of `once_cell` was stabilized in the latest version of Rust, so you might not need the dependency here any more.

Reply:
With the current setup, I think CI is running whatever the latest Rust is at the time. If this becomes a library crate, I guess we'd probably want to have CI run against ever supported Rust version, or at least the oldest and newest?